### PR TITLE
Ticket/master/14 redirected ouptut broken

### PR DESCRIPTION
**Pull request by kbarber**
This solves GH-14 I hope.

### CODE REVIEW

**Comment by luislavena** [comment]
Hmm, question, why adding specs when there are already tests?

**Comment by kbarber** [comment]
@luislavena I wanted unit tests. Your tests are great acceptance tests, but they solve a different problem I figured. I'm happy to also add tests using your style if you like.

**Comment by luislavena** [comment]
@kbarber ignore my stupidity. I just confused win32console with rb-readline were we implemented Test Unit over the acceptance tests.

win32console has no **real** test so your specs are good.

File: tasks/rspec.rake [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Rake tasks to run rspec tests
+
+require 'rubygems'
+require 'rspec'
```
Comment (luislavena) [inline tasks/rspec.rake:4]: If we add rspec dependency we also need to add it to the gemspec:

https://github.com/luislavena/win32console/blob/master/tasks/gem.rake#L22

**Comment by kbarber** [comment]
@luislavena well spotted. I've added those dependencies to commit 39014ea and rebased. Let me know if this doesn't look right.

**Comment by luislavena** [comment]
Hello @kbarber merged this into master with some modifications to rake and spec_helper.

Thank you.

**Pull request merged**